Portugues br.ini

```
@@ -0,0 +1,665 @@
+OmegaMonster = "Metroid Ômega"
+OmegaMonster_Text = "O superpredador do SR388. O Metroid Ômega exibe uma força devastadora; um impacto de suas garras podem aleijar gravemente ou matar animais grandes, e as presas mais protegidas podem ser lançadas pelos ares. #Apesar de sua grande altura, ele pode pular e usar seu peso como arma. As glândulas na boca aquecem a saliva, transformando-a em um lança-chamas orgânico. Sua membrana se tornou muito mais grossa; o uso de poder de fogo máximo é altamente recomendado.# #Essa é a fase final do ciclo de vida de um Metroid, e pouquíssimos atingem essa fase. Seu tamanho é resultado da expansão rápida dos músculos para proporções gigantes de sua forma anterior."
+QueenMonster = "Rainha Metroid"
+QueenMonster_Text = "O único organismo Metroid naturalmente capaz de gerar prole, a Rainha é o coração da espécie e mãe de todos os Metroids nascidos no SR388. Seu papel na colônia prejudicou sua mobilidade, mas desencadeou o desenvolvimento de um pescoço longo e uma cabeça grande para atacar à distância.# #Sua estrutura celular indica sua idade avançada, possivelmente tão antiga quanto o laboratório onde ela se instalou. Os anos reforçaram seu tronco a níveis impenetráveis. #O código genético da Rainha é diferente dos outros Metroids, e ela pode ter atravessado um ciclo de vida singular para atingir essa forma."
+Chozo = "A Raça Perdida"
+Chozo_Text = "Os Chozo foram uma raça aviária antiga e misteriosa. #Incrivelmente avançados, eles colonizaram muitos mundos, espalhando sua mensagem de paz pelo universo. #Existem registros históricos e lendas sobre os Chozo visitarem outras raças com quem compartilharam seu conhecimento e sabedoria. #Esses encontros breves levaram muitas culturas primitivas à sua glória do presente. Dizem que eles ajudaram a consolidar os alicerces da Federação Galáctica.# #O destino dessa raça é desconhecido. Evidências sugerem que eles desapareceram gradativamente do cosmos com o passar dos séculos. #Tudo o que restou em cada planeta que habitaram foram as ruínas de sua então gloriosa civilização."
+GoldenTemple = "O Templo Dourado"
+GoldenTemple_Text = "Este templo fica na base de uma montanha oca. As estátuas, pedras entalhadas e seu projeto sólido confirmam que a estrutura foi construída para ser um santuário, provavelmente com propósito religioso. #O templo resistiu ao teste do tempo, mostrando o conhecimento e experiência de construção dos Chozo.# #Análises detectaram a presença de entidades robóticas estacionárias, colocadas em pontos estratégicos dentro da estrutura. Nenhum sinal de controle foi detectado, essas unidades operam de forma autônoma."
```

Por mas que 'Projeto' seja um tradução de 'Design', eu acho que 'Design' se encaixa melhor aqui.
```suggestion
GoldenTemple_Text = "Este templo fica na base de uma montanha oca. As estátuas, pedras entalhadas e seu design sólido confirmam que a estrutura foi construída para ser um santuário, provavelmente com propósito religioso. #O templo resistiu ao teste do tempo, mostrando o conhecimento e experiência de construção dos Chozo.# #Análises detectaram a presença de entidades robóticas estacionárias, colocadas em pontos estratégicos dentro da estrutura. Nenhum sinal de controle foi detectado, essas unidades operam de forma autônoma."
```